services/gateway/gateway/__main__.py

```diff
@@ -64,7 +64,7 @@ def inventory_get(inventory_id: str):
 
 @app.route('/inventories/<inventory_id>', methods=['PUT'])
 @jwt_required
-def inventory_put():
+def inventory_put(inventory_id: str):
     return rpc.send('Inventory.put', request.json)
```

Comment:
Might be a good idea to merge the `request.json` data and the `inventory_id` somewhat like this:

```
rpc.send('Inventory.put', {
    **request.json,
    'id': inventory_id
})
```

This way you know the ID is always provided via the right channel